### ISSUE TITLE
drake_toppra throwing assert failure

### ISSUE THREAD

**Issue by kamiradi** [moveit/moveit_drake#38]
Hey @sjahr I was trying to generate a few stats for the presentation on the main branch and the drake_toppra plugin seems to be throwing this and crashing the pipeline script. Any clues ?

```
[pipeline_testbench_example-4] abort: Failure at multibody/optimization/toppra.cc:85 in Toppra(): condition 'path.rows() == plant.num_positions()' failed.
```

**Comment by sjahr** [comment]
Yes, we'll need to switch back to panda_arm.urdf in the toppra file for now because the path input transcription needs to be updated.
